test/src/test_pylint.py

```diff
@@ -45,5 +42,14 @@ def test_pylint():
     # Append the checkout-path so all paths are absolute.
     files = map(lambda p: os.path.join(checkout, p), files)
 
-    # Run pylint on all files.
-    subprocess.run(["pylint"] + list(files), check=True)
+    return list(files)
+
+
+def test_pylint(source_files):
+    #
+    # Run `pylint` on all python sources. We simply use `find` to locate
+    # all `*.py` files, and then manually select the reverse-domain named
+    # modules we have.
```

Comment:
The comment is now a bit obsolete, or maybe misleading, as most of that is done by the fixture now, but... also not wrong.

Reply:
Yeah, I did notice and was torn.